config/reports/sameRrnReport.js

```diff
@@ -16,10 +16,14 @@ export default {
       PREFIX adms: <http://www.w3.org/ns/adms#>
 
       SELECT DISTINCT ?person1 ?person2 WHERE {
-        ?person1 a person:Person.
-        ?person2 a person:Person.
-        ?person1 adms:identifier/skos:notation ?rrn.
-        ?person2 adms:identifier/skos:notation ?rrn.
+        GRAPH ?g {
+          ?person1 a person:Person ;
```

Comment:
why do we need this?

Reply:
It seems it gives  different results:

```
      PREFIX person: <http://www.w3.org/ns/person#>
      PREFIX adms: <http://www.w3.org/ns/adms#>
      PREFIX foaf: <http://xmlns.com/foaf/0.1/>
      SELECT DISTINCT ?person1  ?person2 WHERE {

        ?person1 a person:Person.
        ?person2 a person:Person.
        ?person1 adms:identifier/skos:notation ?rrn.
        ?person2 adms:identifier/skos:notation ?rrn.
        FILTER (?person1 != ?person2)
      }
```

Reply:
I am also wondering why they haven't been picked up by the clean up service

Reply:
@cecemel oh good catch, actually I reused the query of the [person-reconciliation-service](https://github.com/lblod/person-reconciliation-service/blob/fcbd3ccdde1d6174ec4813e7c452ea9e94a68843/support.js#L64-L89) to fix the report, so it's normal that the service doesn't pick up anything as the result when specifying the graph is empty.
What that means is that in some cases, the duplicates reside in different graphs, and if we want to merge them we should extend (or update the query) of the reconciliation service